lib/tasks/cortex/core/media.rake

```diff
@@ -17,7 +17,7 @@ namespace :cortex do
 
         puts "Creating Fields..."
 
-        allowed_asset_content_types = %w(txt css js pdf doc docx ppt pptx csv xls xlsx svg ico png jpg gif bmp)
+        allowed_asset_content_types = %w(txt css js pdf doc docx ppt pptx csv xls xlsx zip odt epub svg ico png jpg gif bmp)
```

Comment:
Oh yeah, I think the reason why I added zip has to do with how Paperclip reads `docx` files created on google docs since i think it has some zipped/compressed config docs contained within it. 

But actually I don't think it's necessary for this to work, so I can remove it.